woocommerce/woocommerce
# Shipping Settings: Update shipping settings copy

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Fixes https://github.com/woocommerce/woocommerce/issues/40188
Fixes https://github.com/woocommerce/woocommerce/issues/40191

Updates strings according to TgDI6pJuBmnPyZ810B7uzX-fi



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. See text correclty changed
2. See follow ups outlined in Figma but not addressed here:
* https://github.com/woocommerce/woocommerce/issues/40864
* https://github.com/woocommerce/woocommerce/issues/40890
* https://github.com/woocommerce/woocommerce/issues/40891
* https://github.com/woocommerce/woocommerce/issues/40892



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Updates unreleased feature

</details>
